ledfx/effects/gradient.py

```
@@ -36,6 +36,7 @@ class GradientEffect(Effect):
 
     _gradient_curve = None
     _gradient_roll_counter = 0
+    _gradient_override: "Optional[str]" = None  # venue colour override
```

Flake8 (7.3.0)

[error] undefined name 'Optional'

(F821)